### PR TITLE
feat: Day 2 - MERN Stack, 12 factor app (#39513)

### PR DESCRIPTION
**Pull request by ajayrose25**

### CODE REVIEW

File: mern-stack.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# MERN Stack
+
+## What is the MERN Stack?
+   A stack is the combination of technologies used to create a web application. Any web application will be made using multiple technologies (frameworks, libraries, databases etc).
+
+   The MERN stack is a JavaScript stack that’s designed to make the development process smoother. MERN includes four open-source components: MongoDB, Express, React, and Node.js. These components provide an end-to-end framework for developers to work in.  
+
+### The components of MERN stack are as follows:
```
Comment (VisaaliSuccessive) [inline mern-stack.md]: can be changed to h2 tag

File: mern-stack.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+**2.Express**
+    Express is a web application framework for Node.js, another MERN component. Instead of writing full web server code by hand on Node.js directly, developers use Express to simplify the task of writing server code. There’s no need to repeat the same code over and over, as you would with the Node.js HTTP module.
+
+**3.React**
+    The React library can be used for creating views rendered in HTML. React views are declarative. This means that developers don’t have to worry about managing the effects of changes in the view’s state (the object that determines how components behave) or changes in the data. With React, the same code can run on both the server and the browser.
+
+**4.Node.js**
+    It’s designed to build scalable network applications, and can execute JavaScript code outside of a browser.
+
+#### The Benefits of the MERN Stack
```
Comment (VisaaliSuccessive) [inline mern-stack.md]: can be changed to h2 tag for maintaining consistency

File: mern-stack.md (hunk 1 above)
Comment (ajayrose25) [reply mern-stack.md]: Done

File: mern-stack.md (hunk 2 above)
Comment (ajayrose25) [reply mern-stack.md]: Done

**Pull request merged**